packages/shared/src/components/widgets/PostUsersHighlights.tsx

```diff
@@ -88,7 +103,7 @@ const Image = (props: SourceAuthorProps) => {
   );
 };
 
-const UserHighlight = (props: SourceAuthorProps) => {
+export const UserHighlight = (props: UserHighlightProps): ReactElement => {
```

Comment:
I was confused by this file changes (based on name)
Maybe we should abstract some of these to different components?

(not blocking ofcourse)

Reply:
I thought of it as well as this component is too big and does to many stuff.